pvanalytics/metrics.py

```diff
@@ -36,11 +36,14 @@ def performance_ratio_nrel(poa_global, temp_air, wind_speed, pac, pdc0,
     wind_speed : numeric
         Wind speed at a height of 10 meters [m/s].
 
-    pac : float
+    pac : numeric
         AC power [kW].
 
     pdc0 : float
         Power of the modules at 1000 W/m2 and cell reference temperature [kW].
+        This is otherwise referred to as the DC capacity of the 'array unit'
+        that is associated with the AC power stream, where the array unit could
+        be an inverter or similar.
```

Comment:
I now regret using the phrase "array unit" :P

I think the main source of confusion in the original description is that it talks about modules instead of the array as a whole.  How about revising to just `PV array STC nameplate rating [kW]`?

Reply:
I updated to what you suggest in https://github.com/pvlib/pvanalytics/pull/162/commits/994baa1c393b59f4a54750df99cdaef076383823